cobra/core/Model.py

```diff
@@ -720,3 +654,39 @@ def __exit__(self, type, value, traceback):
         """Pop the top context manager and trigger the undo functions"""
         context = self._contexts.pop()
         context.reset()
+
+
+def get_reactions(model, reactions):
```

Comment:
Awesome. I wrote sooo much code where I have to convert reaction ids to the reactions :+1: